packages/ui/src/machines/authenticator/actors/signIn.ts

```diff
@@ -167,14 +168,13 @@ export function signInActor({ services }: SignInMachineOptions) {
           },
         },
         autoSignIn: {
-          initial: 'submit',
+          initial: 'edit',
           states: {
-            submit: {
+            edit: {
```

Comment:
NIT:  can we remove this nested `edit` state? `autoSignIn`(with code flow)  doesn't have an edit screen to my knowledge.

Reply:
I can rename it to submit, and then have initial go to `submit` instead.

Reply:
Wondering if we can just remove this substate though, since we're only interested in listening to AUTO_SIGN_IN events and are not doing anything with editing or submitting:

```diff
        autoSignIn: {
          initial: 'edit',
-          states: {
-            edit: {
              tags: ['pending'],
              entry: ['clearError', 'sendUpdate'],
              on: {
                AUTO_SIGN_IN: [...]
                AUTO_SIGN_IN_FAILURE: [...\]
              },
            },
            resolved: { always: '#signInActor.resolved' },
            rejected: { always: '#signInActor.rejected' },
-          },
-        },
```

Reply:
wait nvm, I take back what I said, that's not gonna work since there're also `resolved` and `rejected` substates.
 Can we rename this to something like `pending`?

Reply:
OK, I moved it to pending